### DISCUSSION TITLE
wrong SPDX on help pages

### DISCUSSION THREAD

**Discussion by robbiemorrison** [frictionlessdata/frictionlessdata.io#712] [General] [answered]
With reference to this help page: <https://frictionlessdata.io/blog/2018/03/27/applying-licenses/#provide-additional-license-information>

To align with the **SPDX standard**, please correct: " ODC-PDDL-1.0" &rarr; "PDDL-1.0"

More information here: <https://spdx.org/licenses/PDDL-1.0.html>

HTH&nbsp;R

**Comment by lwinfree** [comment, marked as answer]
Thanks Robbie, I've fixed it here: https://github.com/frictionlessdata/website/pull/284 (and feel free to make PRs in the future too if you find other errors! 😄 )
